Jenkinsfile

```diff
@@ -40,21 +40,22 @@ pipeline {
       }
     }
 
-    /* stage('build and publish') { */
-    /*   when { branch 'master' } */
-    /*   steps { */
-    /*     sh 'git config --global user.email james.kassemi+vabot@adhocteam.us' */
-    /*     sh 'git config --global user.name va-bot' */
-    /*     sh 'git config --global credential.helper "/bin/bash ' + env.WORKSPACE + '/scripts/credential-helper.sh"' */
-    /*     withCredentials([[ */
-    /*       $class: 'UsernamePasswordMultiBinding', */
-    /*       credentialsId: 'va-bot', */
-    /*       usernameVariable: 'GIT_USERNAME', */
-    /*       passwordVariable: 'GIT_PASSWORD' */
-    /*     ]]) { */
-    /*       sh 'npm run site' */
-    /*     } */
-    /*   } */
-    /* } */
+    stage('build and publish') {
+      when { branch 'master' }
+      steps {
+        sh 'git config --global user.email james.kassemi+vabot@adhocteam.us'
+        sh 'git config --global user.name va-bot'
+        sh 'git config --global credential.helper "/bin/bash ' + env.WORKSPACE + '/scripts/credential-helper.sh"'
+        withCredentials([[
+          $class: 'UsernamePasswordMultiBinding',
+          credentialsId: 'va-bot',
+          usernameVariable: 'GIT_USERNAME',
+          passwordVariable: 'GIT_PASSWORD'
+        ]]) {
+          sh 'cd packages/documentation'
+          sh 'GITHUB_API_KEY=${env.TOKEN} npm run build'
```

Comment:
Not sure how or if I can test Jenkins locally but I uncommented this and updated the npm commands.